src/Streamly/Internal/Data/Parser.hs

```diff
@@ -560,17 +573,17 @@ escapedSliceSepBy _cond _esc = undefined
 --
 -- For example,
 --
--- >>> escapedFrameBy (== '{') (== '}') (== '\\') S.toList $ S.fromList "{hello}"
--- > "hello"
+-- > Stream.parse (Parser.escapedFrameBy (== '{') (== '}') (== '\\') Fold.toList) $ Stream.fromList "{hello}"
```

Comment:
Examples that do not work or use unimplemented functions had their markup changed from `>>>` to `>` but this results in  haddock that looks like this
![image](https://user-images.githubusercontent.com/15224715/107344275-49ea9780-6ae8-11eb-8902-a7560087e52a.png)

It's probably better to surround them in `@` instead.